dashing/distribution.yaml

```diff
@@ -1261,6 +1262,7 @@ repositories:
       url: https://github.com/ros2-gbp/libyaml_vendor-release.git
       version: 1.0.0-1
     source:
+      test_abi: true
```

Comment:
This one doesn't have `test_pull_requests` or `test_commits` enabled, so does `test_abi` have any effect?

Reply:
test_commits defaults to true for Dashing builds: https://github.com/ros2/ros_buildfarm_config/blob/b90b4b332131c4e8b66ad1784cf00b9d16a71825/dashing/source-build.yaml#L91-L94